files/template.html

```diff
@@ -11,6 +11,7 @@
     <header>
       <div>
         <h1>Content Headline</h1>
+        <small><time>{{ publishDate }}</time></small>
```

Comment:
@nachtfunke I really like this addition of functionality! However I'm going to remove this from the main HTML template to prevent having a "breaking change", but it was very useful for testing/validating.
```suggestion
```